dd-sdk-android/src/main/json/trace/span-schema.json

```diff
@@ -0,0 +1,210 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "span-schema.json",
+  "title": "SpanEvent",
+  "type": "object",
+  "description": "Schema of a Span Event",
+  "required": [
+    "trace_id",
+    "span_id",
+    "parent_id",
+    "resource",
+    "name",
+    "service",
+    "duration",
+    "start",
+    "error",
+    "type",
+    "meta",
+    "metrics"
+  ],
+  "properties": {
+    "trace_id": {
+      "type": "string",
+      "description": "The id of the trace this Span belongs to",
+      "readOnly": true
+    },
+    "span_id": {
+      "type": "string",
+      "description": "The unique id of this Span",
+      "readOnly": true
+    },
+    "parent_id": {
+      "type": "string",
+      "description": "The id of the parent Span in this trace",
+      "readOnly": true
+    },
+    "resource": {
+      "type": "string",
+      "description": "The resource name"
+    },
+    "name": {
+      "type": "string",
+      "description": "The name of this Span"
+    },
+    "service": {
+      "type": "string",
+      "description": "The service name"
+    },
+    "duration": {
+      "type": "integer",
+      "description": "The duration of this Span in nanoseconds",
+      "readOnly": true
+    },
+    "start": {
+      "type": "integer",
+      "description": "The Span start time in nanoseconds",
+      "readOnly": true
+    },
+    "error": {
+      "type": "integer",
+      "description": "Span error flag. If 1 that means there was an error thrown during the duration of the Span",
+      "readOnly": true,
+      "default": 0
+    },
+    "type": {
+      "type": "string",
+      "description": "The type of the Span. For Mobile this will always be 'CUSTOM'",
+      "const": "custom"
+    },
+    "metrics": {
+      "type": "object",
+      "description": "The metrics data of this Span event",
+      "properties": {
+        "_top_level": {
+          "type": "integer",
+          "description": "Top level flag. If 1 means that this Span is the root of this Trace",
+          "readOnly": true
+        }
+      },
+      "additionalProperties": {
+        "type": "number",
+        "description": "additional metrics"
+      }
+    },
+    "meta": {
+      "type": "object",
+      "description": "The metadata of this Span event",
+      "properties": {
+        "version": {
+          "type": "string",
+          "description": "The client application package version",
+          "readOnly": true
+        },
+        "_dd": {
+          "type": "object",
+          "properties": {
+            "source": {
+              "type": "string",
+              "description": "The trace source",
+              "const": "android"
```

Comment:
this is not going to be a const, it can be changed by the bridge. But let's make it readonly